apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/Aggregations.cs

```diff
@@ -32,4 +32,56 @@ public static class Aggregations
     /// <returns>A count(*) aggregation.</returns>
     public static Aggregation Count(string alias = "") =>
         new Aggregation { Count = new Count(), Alias = alias };
+
+    /// <summary>
```

Comment:
I'm slightly surprised we don't have a unit test for Count already. Given how trivial each of these methods is, I'm not overly bothered - but it wouldn't hurt to have a couple of unit tests for each method (with and without alias, basically).

Reply:
do think that for the trivial amount that we have added in client libraries for this, the integration tests already cover everything that needs to be tested and we shouldn't just be adding more test for the sake of it, as is already done in count.

Reply:
I'm not going to insist on it, but I do generally like to see unit tests for things that are *so* easy to test like this. (See FilterTest for examples. I'm happy with light integration test coverage *because* we generally have good unit test coverage - meaning that integration tests can really be about *integration*: "I'm confident in the proto I'm sending due to the unit tests, now I can test that that has the effect I expect."

Reply:
I have to agree with Jon here, in particular with:

> I'm happy with light integration test coverage because we generally have good unit test coverage- meaning that integration tests can really be about integration

For instance, with no unit tests, if the integration test start failing, it's harder to know whether that's a service failure or our own code failure. In general, we should have thorough unit testing and only representative integration testing.